src/demux/mp3demuxer.ts

```diff
@@ -52,6 +52,9 @@ class MP3Demuxer extends BaseAudioDemuxer {
   }
 
   appendFrame (track, data, offset) {
+    if (this.initPTS === null) {
+      return;
+    }
```

Comment:
It might be a good idea to error here. If you we assert that it's null, it will only error in debug. In production builds where we strip out `console.assert` the null value would get through though.

```suggestion
    console.assert(this.initPTS !== null);
```